std/node/buffer.ts

```diff
@@ -4,13 +4,22 @@ import * as base64 from "../encoding/base64.ts";
 import { Encodings, normalizeEncoding, notImplemented } from "./_utils.ts";
 
 const notImplementedEncodings = [
-  "ascii",
   "binary",
   "latin1",
   "ucs2",
   "utf16le",
 ];
 
+function decodeAscii(buffer: Uint8Array) {
+  return Array.from(buffer).map((byte) => String.fromCharCode(byte & 0x7F)).join("");
+}
+
+function encodeAscii(text: string) {
+  return new Uint8Array(
+    Object.keys(text).map((i) => text.charCodeAt(Number(i))),
```

Comment:
If performance matters here: there's probably a steep performance cliff for longer strings.

V8 has a small-numbers-as-strings cache that's not effective for longer strings and I doubt V8 can figure out it doesn't actually need to materialize the indexes as strings.